CAIPs/caip-25.md

```diff
@@ -61,61 +61,175 @@ If a connection is rejected, the wallet MAY respond with a generic error or sile
 
 #### Request
 
+```typescript
+interface CAIP25JsonRpcRequest {
+  id: number;
+  jsonrpc: "2.0";
+  method: "wallet_createSession";
+  params: {
+    scopes: {
+      [scopeKey: string]: {
+        chains?: string[];
+        methods: string[];
+        notifications: string[];
+      };
```

Comment:
i think it would be a good idea to include the original `accounts` property here

Reply:
Agreed w/ Jiexi ☝️ we use this property in our current implemenation

Reply:
my preference is still
mandatory `accounts` array but can be empty > optional `accounts` array > no accounts array allowed in request